packages/dev-plugin/src/index.ts

```diff
@@ -0,0 +1,225 @@
+import fs from 'fs';
+// import os from 'os';
+// import shorthash from 'shorthash';
+import chalk from 'chalk';
+import { ensureTempDir, isDev } from './utils';
+import { type DevWorker, createDevWorker } from '@module-federation/dev-kit';
+import {
+  moduleFederationPlugin,
+  normalizeOptions,
+} from '@module-federation/sdk';
+import {
+  WEB_CLIENT_OPTIONS_IDENTIFIER,
+  WebClientOptions,
+} from '@module-federation/dev-server';
+import type { Compiler, WebpackPluginInstance } from 'webpack';
+import path from 'path';
+
+const MF_DEV_PLUGIN = 'MFDevPlugin';
```

Comment:
you don't need this constant, the definition inside the class is more than enough